src/main/java/com/acrolinx/sidebar/utils/MatchUtils.java

```diff
@@ -51,6 +52,23 @@ public static List<? extends AbstractMatch> filterDangerousMatches(
         .collect(Collectors.toList());
   }
 
+  public static List<? extends AbstractMatch> filterInsignificantWhitespaceMatches(
+      List<? extends AbstractMatch> abstractMatches, String lastCheckedContent) {
+    return abstractMatches.stream()
+        .filter(
+            match -> {
+              int startOffset = match.getRange().getMinimumInteger();
```

Comment:
for better readability, extract match.getRange() into a local variable to avoid calling it twice.

Reply:
Fixed